CBI.Jenkinsfile

```diff
@@ -123,19 +123,29 @@ spec:
           envName = ' (JIRO)'
         }
         
-        def curResult = currentBuild.currentResult
+        def sendNotification = true
         def color = '#00FF00'
-        if (curResult == 'SUCCESS') {
-           if (currentBuild.previousBuild != null && currentBuild.previousBuild.result != 'SUCCESS') {
-             curResult = 'FIXED'
-           }
+        def curResult = currentBuild.currentResult
+        def lastResult = 'NONE'
+        if (currentBuild.previousBuild != null) {
+          lastResult = currentBuild.previousBuild.result
+        }
+        if (lastResult == 'NONE') {
+          curResult = "NEW: ${curResult}"
```

Comment:
What will this do when failing at the first time

Reply:
This part is only executed in the first build. If the first build is red, you will get a "NEW: FAILURE".